integrationtest/integration_test.go

```diff
@@ -138,69 +124,6 @@ func TestDefineTask(t *testing.T) {
 	if state := tsr.Status.State; state != "unscheduled" {
 		t.Errorf("Expected 'state' to be 'unscheduled', but got %s", state)
 	}
-	submittedPayload := cs.HttpRequestBody
-
-	// only the contents is relevant below - the formatting and order of properties does not matter
-	// since a json comparison is done, not a string comparison...
-	expectedJson := []byte(`
-	{
-	  "created":  "` + created.UTC().Format("2006-01-02T15:04:05.000Z") + `",
-	  "deadline": "` + deadline.UTC().Format("2006-01-02T15:04:05.000Z") + `",
-	  "expires":  "` + expires.UTC().Format("2006-01-02T15:04:05.000Z") + `",
-
-	  "taskGroupId": "` + taskGroupID + `",
-	  "workerType":  "win2008-worker",
-	  "schedulerId": "go-test-test-scheduler",
-
-	  "payload": {
-	    "features": {
-	      "relengApiProxy":true
-	    }
-	  },
-
-	  "priority":      "high",
-	  "provisionerId": "win-provisioner",
-	  "retries":       5,
-
-	  "routes": [
-	    "tc-treeherder.mozilla-inbound.bcf29c305519d6e120b2e4d3b8aa33baaf5f0163",
-	    "tc-treeherder-stage.mozilla-inbound.bcf29c305519d6e120b2e4d3b8aa33baaf5f0163"
-	  ],
-
-	  "scopes": [
-	  	"queue:task-priority:high"
-	  ],
-
-	  "tags": {
-	    "createdForUser": "cbook@mozilla.com"
-	  },
-
-	  "extra": {
-	    "index": {
-	      "rank": 12345
-	    }
-	  },
-
-	  "metadata": {
-	    "description": "Stuff",
-	    "name":        "[TC] Pete",
-	    "owner":       "pmoore@mozilla.com",
-	    "source":      "http://everywhere.com/"
-	  }
-	}
-	`)
-
-	jsonCorrect, formattedExpected, formattedActual, err := jsontest.JsonEqual(expectedJson, []byte(submittedPayload))
-	if err != nil {
-		t.Fatalf("Exception thrown formatting json data!\n%s\n\nStruggled to format either:\n%s\n\nor:\n\n%s", err, string(expectedJson), submittedPayload)
-	}
-
-	if !jsonCorrect {
-		t.Log("Anticipated json not generated. Expected:")
-		t.Logf("%s", formattedExpected)
-		t.Log("Actual:")
-		t.Errorf("%s", formattedActual)
-	}
```

Comment:
I don't want to delete this test. Please use `APICall` directly instead, and let's create a different test that uses a generated method (or maybe there is already one).